core/modules/update/theme/update.theme.inc

```diff
@@ -105,7 +108,12 @@ function theme_update_report($variables) {
 
     $row = '<div class="version-status">';
     $status_label = theme('update_status_label', array('status' => $project['status']));
-    $row .= !empty($status_label) ? $status_label : check_plain($project['reason']);
+    if (empty($status_label)) $status_label = check_plain($project['reason']);
```

Comment:
Coding standards: We always include squiggle brackets around IF statements. https://api.backdropcms.org/php-standards#control-structures